rde/device.cpp

```diff
@@ -80,6 +80,14 @@ void Device::refreshDeviceInfo()
     deviceUpdated();
 }
 
+void Device::performRDEOperation(struct OperationInfo op_info)
```

Comment:
Please make sure this object should get deleted once the operation task completed.